src/resolvers/userResolver.ts

```diff
@@ -1001,8 +1001,9 @@ const resolvers: any = {
           expiresIn: '2d',
         })
         const newToken: any = token.replaceAll('.', '*')
+        const deepLink = `${process.env.FRONTEND_LINK}/redirect/?path=/auth/forget/new-password&token=${newToken}`
```

Comment:
The auth route format is wrong. 

Example of possible routes:
- /auth/reset-password
- /auth/forgot-password